server/src/test/java/org/opensearch/extensions/ExtensionsManagerTests.java

```diff
@@ -115,15 +115,15 @@ public class ExtensionsManagerTests extends OpenSearchTestCase {
         "     hostAddress: '127.0.0.0'",
         "     port: '9300'",
         "     version: '0.0.7'",
-        "     opensearchVersion: '3.0.0'",
-        "     minimumCompatibleVersion: '3.0.0'",
+        "     opensearchVersion: '" + Version.CURRENT.toString() + "'",
+        "     minimumCompatibleVersion: '" + Version.CURRENT.toString() + "'",
```

Comment:
I think for tests we want to use the minimum as the major.0.0, otherwise this perpetuates the idea that the extension is narrowly compatible with the current version only, no?

Reply:
Hi @dblock ,

Are you saying that we should we have different version  like 2.0.0 for first extension and 3.0.0 or 4.0.0 for second extension and so on ? meaning tests has to pass for extensions of all version otherwise we are saying tests are compatible for only the current versions and not the ones before ?

Thanks and Regards
Varun

Reply:
Exactly. Have predictable examples plus one for current version.